File: 1/solution_1.rs
fn sum_multiples() -> u32{
    const _THRESHOLD: u32 = 1000;
    let mut sum = 0u32;
    for i in 1.._THRESHOLD{
       if i % 3 == 0 || i % 5 == 0{
            sum += i
       }
    }
    sum

}

fn main(){
    println!("{}", sum_multiples())
    
}

File: 1/solution_1.go
package main

import "fmt"

func sumMultiples() int {
	const threshold = 1000
	var sum = 0
	for i := 0; i < threshold; i++ {
		if i% 3 == 0 || i%5 == 0 {
			sum += i
		}
	}
	return sum 
}

func main(){
	fmt.Println(sumMultiples())
}